### ISSUE TITLE
have nsqd client re-fresh the IP address for lookupd connections and connect to new lookupd's on IP changes

### ISSUE THREAD

**Issue by danbf** [nsqio/nsq#1233]
right now an nsqd  client doesn't refresh it's ip connections to lookupd on lookupd DNS changes.  this causes a problem on long running instances when a lookupd instance gets rebuilt.

**Comment by starsz** [comment]
Hello，I think you can config the lookupd address by using IP.
When a lookupd instance gets rebuilt, you can update by using `PUT /config/nsqlookupd_tcp_addresses` interface. 

See nsq doc https://nsq.io/components/nsqd.html#put-confignsqlookupd_tcp_addresses.

**Comment by mreiferson** [comment]
Agreed, although @starsz points out a workaround, I think it's a reasonable expectation that if you use DNS to provide some operational control over `nsqlookupd` deployments that it should generally work as expected.

**Comment by starsz** [comment]
Aha，@mreiferson  I think I can contribute to this project.  : )

**Comment by mreiferson** [comment]
@starsz please do!
